Library/DiscUtils.Core/Internal/Utilities.cs

```diff
@@ -281,6 +281,9 @@ public static string ResolveRelativePath(string basePath, string relativePath)
                 return relativePath;
             }
 
+            if (!basePath.EndsWith(@"\"))
+                basePath = Path.GetDirectoryName(basePath);
+
```

Comment:
I can see other parts of this code works on the "\"-premise, but for new code we need to be careful.